packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -645,6 +645,7 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
             let reqStToRespEndTime: number | undefined; // responseEnd - requestStart
             let networkTime: number | undefined; // responseEnd - startTime
             const spReqDuration = response.headers.get("sprequestduration");
+            const msEdge = response.headers.get("x-msedge-ref");
 
             // getEntriesByType is only available in browser performance object
             const resources1 = performance.getEntriesByType?.("resource") ?? [];
```

Comment:
question more for vlad (probably): is there a way to dump this telemetry values if forcedebug mode is turned on in the debugger?

Reply:
You can use localStorage.debug = "fluid:telemetry:*" to see have all telemetry dumped to console